modules/openshift-ai-connector-for-rhdh/proc-setting-up-openshift-ai-connector-for-rhdh-with-rhoai.adoc

```diff
@@ -0,0 +1,146 @@
+:_mod-docs-content-type: PROCEDURE
+
+[id="proc-setting-up-openshift-ai-connector-for-rhdh-with-rhoai_{context}"]
+= Setting up {openshift-ai-connector-name} with {rhoai-brand-name}
+
+The installation of the {openshift-ai-connector-name} requires manual updates to {product-very-short}-related Kubernetes resources.
+
+.{rhoai-short} Prerequisites
+
+* To have Model Cards from the Model Catalog imported as TechDocs, you must use {rhoai-short} version 2.25 or later, and the *Model Catalog* dashboard and a Model Registry need to be enabled (they are both disabled by default).
```

Comment:
```suggestion
* To import model cards from the model catalog as TechDocs, you must use {rhoai-short} version 2.25 or later, and you must ensure that the model registry component is enabled. For more details, see link:https://docs.redhat.com/en/documentation/red_hat_openshift_ai_self-managed/2.25/html-single/enabling_the_model_registry_component/index[Enabling the model registry component].
```

Reply:
Note that model registry, model catalog, and model cards are all lowercase terms in RHOAI documentation.